cloud/deployment/deployment.go

```diff
@@ -195,39 +226,24 @@ func Create(label, workspaceID, description, clusterID, runtimeVersion, dagDeplo
 	if err != nil {
 		return err
 	}
-
-	configOption, err := client.GetDeploymentConfig()
-	if err != nil {
-		return err
-	}
-
-	if schedulerAU == 0 {
-		schedulerAU = configOption.Components.Scheduler.AU.Default
-	}
-
-	if schedulerReplicas == 0 {
-		schedulerReplicas = configOption.Components.Scheduler.Replicas.Default
+	coreDeploymentType := astrocore.GetDeploymentOptionsParamsDeploymentType(deploymentType)
+	var coreCloudProvider astrocore.GetDeploymentOptionsParamsCloudProvider
+	if cloudProvider == awsCloud {
```

Comment:
do we need something for azure?

Reply:
Yes, we were supposed to add this. Let's do that in this PR

Reply:
okay added